packages/hub/src/session.ts

```diff
@@ -21,15 +21,19 @@ export class Session implements SessionInterface {
   public release?: string;
   public sid: string = uuid4();
   public did?: string;
-  public timestamp: number = Date.now();
-  public started: number = Date.now();
-  public duration: number = 0;
+  public timestamp: number;
+  public started: number;
```

Comment:
Considering the absence of a better type, and that changing the name here might be too much, what do you think about adding a comment stating these are seconds since the UNIX epoch?

Reply:
Great idea, I will add it to the `SessionContext` type.